admin_app_2/src/app/add-faq/page.tsx

```diff
@@ -0,0 +1,174 @@
+"use client";
+import React from "react";
+import { useSearchParams } from "next/navigation";
+import {
+  Typography,
+  Button,
+  Chip,
+  Input,
+  TextField,
+  Switch,
+  FormControlLabel,
+  Tooltip,
+} from "@mui/material";
+import { Layout } from "@/components/Layout";
+import {
+  Add,
+  AddCircle,
+  Check,
+  CheckBox,
+  ChevronLeft,
+  Info,
+  InfoOutlined,
+  Label,
+} from "@mui/icons-material";
+import { appColors, appStyles, sizes } from "@/utils";
+import LanguageButtonBar from "@/components/LanguageButtonBar";
+
+const AddFAQPage = () => {
+  const searchParams = useSearchParams();
+  const contentID = searchParams.get("contentID") ?? "";
+
+  return (
+    <Layout.FlexBox flexDirection={"column"} sx={{ p: sizes.doubleBaseGap }}>
+      <Header contentID={contentID} />
+      <Layout.FlexBox
+        flexDirection={"column"}
+        sx={{ px: sizes.doubleBaseGap, mx: sizes.smallGap }}
+      >
+        <Layout.Spacer multiplier={2} />
+        <TopicRow />
+
+        <Layout.Spacer multiplier={2} />
+        <ContentBox />
+
+        <Layout.Spacer multiplier={2} />
+
+        <Layout.Spacer multiplier={2} />
```

Comment:
Remove duplication and set size to 1
```suggestion
        <Layout.Spacer multiplier={1} />
```